### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by Cheechoz** [Tobi823/ffupdater#249]
Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.InvalidApiResponseException: can't find release after all tries - abort
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer.updateCheck(SourceFile:2)
	at de.marmaro.krt.ffupdater.network.github.GithubConsumer$updateCheck$1.invokeSuspend(Unknown Source:19)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:82)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.3.0 (114) release |
| Device | SM-S908U (b0qsqw, b0q, taro) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 12 (SDK: 31) |
| OS | SWDK6014, dpi, release-keys, 1663545410000 |

**Comment by Tobi823** [comment]
Fixed version (77.3.1) is now released

**Comment by Cheechoz** [comment]
Just updated the app and all is smooth again , my thanks to you 😀👍

Sent from my T-Mobile 5G Device
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
